src/js/media-poster-image.js

```diff
@@ -35,7 +40,7 @@ const setBackgroundImage = (el, image) => {
 
 class MediaPosterImage extends window.HTMLElement {
   static get observedAttributes() {
-    return ['placeholder-src', 'src'];
+    return [Attributes.PLACEHOLDER_SRC, Attributes.SRC];
```

Comment:
non-blocker: alternative is to do `Object.values(Attributes)`

Reply:
decided to not use values just because that's "incidental" and not "definitional" (aka all of the values in `Attributes` _**happen to**_ be `observedAttributes`, but that could change and isn't inherent to their relationship).

Reply:
Ah, right. Worked here incidentally. Wasn't a blocker or necessary anyway.